bin/http-server

```diff
@@ -50,6 +51,7 @@ var port = argv.p || parseInt(process.env.PORT, 10),
     ssl = !!argv.S || !!argv.ssl,
     proxy = argv.P || argv.proxy,
     utc = argv.U || argv.utc,
+    baseDir = argv.b,
```

Comment:
This will need to change to `argv.get('baseDir')`, and probably use an alias for `b` after #131 merges.

For now, it should include `|| argv.baseDir`, I think, right?